VS2013/SetVersion.bat

```diff
@@ -0,0 +1,3 @@
+rem pyi-set_version is a Python tool (part of PyInstaller library) to embed the version information in the DLL
```

Comment:
I can send you the exe if you want it, I can't find a direct download without installing PyInstaller.